accessibility_scanning/README.md

```diff
@@ -0,0 +1,131 @@
+## Accessibility Scanning Using Pa11y-ci
+
+### Introduction
+
+Building a website or application that is easy accessible toeveryone is  not only an important of the user experience, but also a requirement of all federally funded projects. GSA provides a helpful [Section 508](https://section508.gov)  portal. [Pa11y](https://github.com/pa11y/pa11y) is an automation tool that helps you scan your static web pages to check for accessibility problems and errors. It can be setup on your machine locally or remotely using a CI tool.
+
+### Installation and setup(For MacOS)
```

Comment:
I might update the organization of this file a bit to be (this is just the headers, I'd put in your existing content between them:

```
## Running Pa11y locally

### Installation and setup (For MacOS)
### Running Pa11y-ci on your local machine

## Running Pa11y-ci with CI

### Circle CI setup instructions
### Travis CI setup instructions (to be written)

```